examples/internal/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -160,6 +160,26 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 			}
 		}
 	}
+	responses: {
+		key: "500";
+		value: {
+			description: "Server error";
+			headers: {
+				key: "x-correlation-id"
+				value: {
+					description: "Unique event identifier for server requests\n\nExample: `2438ac3c-37eb-4902-adef-ed16b4431030`\n\nPattern: `^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$`"
+					type: "string"
+					format: "uuid"
+					default: "\"2438ac3c-37eb-4902-adef-ed16b4431030\""
```

Comment:
Can we make this test all the options we allow you to specify? E.g. add pattern too.

Reply:
done added pattern and example